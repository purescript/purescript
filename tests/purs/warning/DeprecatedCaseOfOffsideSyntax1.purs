-- @shouldWarnWith WarningParsingModule
module DeprecatedCaseOfOffsideSyntax1 where

data Foo = Foo Int

test :: Foo -> Int
test = case _ of
    Foo i ->
  i
