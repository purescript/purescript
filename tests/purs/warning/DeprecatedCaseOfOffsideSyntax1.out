Warning found:
at tests/purs/warning/DeprecatedCaseOfOffsideSyntax1.purs:9:3 - 9:4 (line 9, column 3 - line 9, column 4)

  Dedented expressions in case branches are deprecated and will be removed in a future release. Indent the branch's expression past it's binder instead.


See https://github.com/purescript/documentation/blob/master/errors/WarningParsingModule.md for more information,
or to contribute content related to this warning.

